acoustid/data/stats.py

```
@@ -4,7 +4,7 @@
 import datetime
 import logging
 import urllib.parse
-from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
+from typing import Any, Iterable, Optional, Tuple, Union
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)